docs/bridge/spec.md

```diff
@@ -0,0 +1,991 @@
+trust on the locking-chain, and a regular XRP ledger account is used to issue
+assets on the issuing-chain. These accounts will have their master keys disabled
+and funds will move from these accounts through a new set of transactions
+specifically meant for cross-chain transfers. A special set of "witness servers"
+are used to prove that assets were put into trust on the locking-chain or
+returned on the issuing-chain. A new ledger object called a "xchain claim id" is
+used to prevent transaction replay.
+
+A source chain is the chain where the cross-chain transfer starts - by either
+putting assets into trust on the locking-chain or returning wrapped assets on
+the issuing-chain. The steps for moving funds from a source chain to a
+destination chain are:
+
+1) On the destination chain, an account submits a transaction that adds a ledger
+   object called a "xchain claim id" that will be used to identify the
+   initiating transaction and prevent the initiating transaction from being
+   claimed on the destination chain more than once. This transaction will
+   include a signature reward amount, in XRP. Reward amounts must match the
+   amount specified by the bridge ledger object, and the reward amount will
+   be deducted from the account's balance and held on this ledger object.
+   Collecting rewards is discussed below. The door account will keep a new
+   ledger object - a "bridge". This "bridge" ledger object will keep a counter
+   that is used for these "xchain claim ids". A "xchain claim id" will be
+   checked out from this counter and the counter will be incremented. Once
+   checked out, the claim id would be owned by the account that submitted the
+   transaction. See the section below for what fields are present in the new
+   "bridge" ledger object and "xchain claim id" ledger object. The actual number
+   must be retrieved from the transaction metadata on a validated ledger.
+   
+2) On the source chain, an initiating transaction is sent from a source account.
+   This transaction will include the amount to transfer, bridge spec, "xchain
+   claim id" from step (1), and an optional destination account on the
+   destination chain. The asset being transferred cross-chain will be
+   transferred from the source account to the door account.
+   
+3) When a witness server sees a new cross-chain transaction, it submits a
+   transaction on the destination chain that adds a signature attesting to the
+   cross-chain transaction. This will include the amount being transferred
+   cross-chain, the account to send the signature reward to on the destination
+   chain, the bridge spec, the sending account, and the optional destination
+   account. These signatures will be accumulated on the "xchain claim id" object
+   on the destination chain. The keys used in these signatures must match the
+   keys on the multi-signers list on the door account.
+
+4) When a quorum of signatures have been collected, the cross-chain funds can be
```

Comment:
```suggestion
4) When a quorum of signatures have been collected, the cross-chain transfer's funds may be
```
nit: Could be more specific

Reply:
Thanks @wojake. This spec is out of date. I removed it from the patch. The new spec will be folded in after the discussion on the XRPL repo.

Reply:
If you don't mind, I'm going to close this as resolved, but if you'd like to reopen to discuss further I don't object.